cmd/coder/urls.go

```diff
@@ -137,35 +223,27 @@ func (sub delSubCmd) Run(fl *pflag.FlagSet) {
 		exitUsage(fl)
 	}
 
-	if !portIsValid(port) {
+	portNum, err := validatePort(port)
+	if err != nil {
 		exitUsage(fl)
 	}
```

Comment:
Hmm. I wonder if this and `accessLevelIsValid()` should then just call `exitUsage()` and `log.Fatal()` themselves rather than `log.Error()` and relying on the caller to exit. What's your preference on exit/fatal from 'helper' funcs?